notebooks/01-visualize-amounts-bar-plots.ipynb

```diff
@@ -0,0 +1,1384 @@
+{

```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from bokeh.plotting import figure, show
</pre><p>suggest splitting this into its own cell. Then the cells for vertical and horizontal plots can have parallel structure, starting with the same comment <code>"# plot a ... bar"</code>  I'd suggest making the other top-level comments the same as well. </p><p> </p><p>The great benefit of developing parallel structure between things for readers is that he helps them compare closely and see where there the differences.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokeh/dataviz-fundamentals/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>